app/screens/send-bitcoin-screen/send-bitcoin-screen.tsx

```diff
@@ -600,188 +604,209 @@ export const SendBitcoinScreenJSX: ScreenType = ({
   reset,
   prefCurrency,
   nextPrefCurrency,
-}: SendBitcoinScreenJSXProps) => (
-  <Screen preset="fixed">
-    <ScrollView
-      style={styles.mainView}
-      contentContainerStyle={{ justifyContent: "space-between" }}
-    >
-      <View style={styles.section}>
-        <InputPayment
-          prefCurrency={prefCurrency}
-          nextPrefCurrency={nextPrefCurrency}
-          editable={
-            paymentType === "lightning" || paymentType === "onchain"
-              ? amountless && (status === "idle" || status === "error")
-              : status !== "success" // bitcoin // TODO: handle amount properly
-          }
-          initAmount={initAmount}
-          onUpdateAmount={(input) => {
-            setAmount(input)
-            setStatus("idle")
-          }}
-          forceKeyboard
-          price={price}
-        />
-      </View>
-      <View style={{ marginTop: 18 }}>
-        <Input
-          placeholder={translate("SendBitcoinScreen.input")}
-          leftIcon={
-            <View style={styles.row}>
-              <Text style={styles.smallText}>{translate("common.to")}</Text>
-              <Icon
-                name="ios-log-out"
-                size={24}
-                color={color.primary}
-                style={styles.icon}
-              />
-            </View>
-          }
-          onChangeText={setDestination}
-          rightIcon={
-            UsernameValidation.hasValidLength(destination) &&
-            !potentialBitcoinOrLightning &&
-            paymentType === "username" ? (
-              loadingUserNameExist ? (
-                <ActivityIndicator size="small" />
-              ) : UsernameValidation.isValid(destination) && usernameExists ? (
-                <Text>✅</Text>
+}: SendBitcoinScreenJSXProps) => {
+  const [isDestinationFocused, setIsDestinationFocused] = useState(false)
+  const [isMemoFocused, setIsMemoFocused] = useState(false)
+  const [isFeeFocused, setIsFeeFocused] = useState(false)
```

Comment:
I would consider creating a Galoy specific Input component and encapsulate this type of logic within it. `input-payment.tsx` could also use that as a base.

Reply:
I agree. Do you like the design of the underline? If so, I can work on encapsulating the input.